baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/RedundantModifier.java

```diff
@@ -41,7 +42,7 @@
         link = "https://github.com/palantir/gradle-baseline#baseline-error-prone-checks",
         linkType = BugPattern.LinkType.CUSTOM,
         providesFix = BugPattern.ProvidesFix.REQUIRES_HUMAN_ATTENTION,
-        severity = BugPattern.SeverityLevel.SUGGESTION,
+        severity = SeverityLevel.WARNING,
```

Comment:
seems like this should be an error

Reply:
I think we want WARN here, iirc we use a flag to ignore error-prone warnings in generated code but it doesn't apply to errors.

Reply:
I believe we explicitly opt out of errorprone on generated source sets: https://github.com/palantir/gradle-baseline/blob/a63a0adcf47044bdaae2f8a2c453159ad347ed02/gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java#L210

Reply:
Can we change this in a FLUP? I'd prefer not to accidentally break things in this PR.

Reply:
+1 for handling warn->error separately. I don't think our regex catches 100% of generated code and I'm not sure it's worth the risk breaking some folks in order to fail builds in repos that don't fail on errors